### PR TITLE
added export and import as base64 string function

### PR DESCRIPTION
**Pull request by AlexanderProd**
Hi, 

Here's the new version of import and export as base64 string. Hope this is fine.

### CODE REVIEW

**Comment by AlexanderProd** [comment]
@ahmdrz Why is the build failing? Doesn't seem to be my code, right?

**Comment by ahmdrz** [comment]
Yes, it's not about your code.

File: utilities/import.go [hunk 1]
```diff
@@ -2,10 +2,22 @@ package utilities
...
 	"github.com/ahmdrz/goinsta"
 )
 
 func ImportFromBytes(inputBytes []byte) (*goinsta.Instagram, error) {
 	return goinsta.ImportReader(bytes.NewReader(inputBytes))
 }
+
+// ImportFromBase64String imports instagram configuration from a base64 encoded string.
+//
+// This function does not set proxy automatically. Use SetProxy after this call.
```
Comment (ahmdrz) [inline utilities/import.go]: Very nice :+1:

**Comment by AlexanderProd** [comment]
Can you create a new release with the addition?

**Comment by ahmdrz** [comment]
Okay, now V2.4.2 is ready to use.

**Pull request merged**